packages/discovery-provider/src/queries/get_track_comment_count.py

```diff
@@ -52,6 +64,7 @@ def _get_counts(session):
                     and_(
                         CommentReport.user_id != current_user_id,
                         CommentReport.user_id != track.c.owner_id,
+                        ~Comment.comment_id.in_(high_karma_reporters),
```

Comment:
oh that's cool is that negation?

Reply:
yea its a not

Reply:
there should be a not_in or notin_ operator which might be clearer

Reply:
ah yeah that would be better